src/getting_started/cli.md

```diff
@@ -0,0 +1,105 @@
+# Command Line Tool: nargo
```

Comment:
Most of this section details how to setup and install nargo. I'd rather it stay titled "Installing Noir" or "Installing Nargo" since I think it'd be less confusing for users trying to figure out where the directions to install it are.

Reply:
Good idea. "Installing Noir" could be confusing now that we also offer the TypeScript wrapper.
See if separating Nargo into a new section like in the latest commit looks more intuitive?